docs/docusaurus/docs/core/define_expectations/create_an_expectation.md

```diff
@@ -40,7 +40,7 @@ An Expectation is a verifiable assertion about your data. Expectations make impl
    from great_expectations import expectations as gxe
    ```
 
-2. Determine the Expectation's parameters
+2. Determine the Expectation's required parameters
```

Comment:
**Note for reviewer** I added this for a more clear contrast with "3. Optional. Determine the Expectation's other parameters"